### PR TITLE
[EWL-4487] SG2 | Create Information Card molecule and Partner Information molecule

### PR DESCRIPTION
**Pull request by labbydev**
## Ticket(s)
**Jira Ticket**
- [EWL-4487: SG2 | Create Information Card molecule and Partner Information organism](https://issues.ama-assn.org/browse/EWL-4487)

## Description
* Create the "Information Card" molecule
* Create the "Info Card as Media Contact" variant
* Create the "Partner Information" molecule
* Create the "Partner List" organism
* Add the "Partner List" organism to the "Press Release" page

_This most definitely should have been multiple tickets. Bust since the data structure changed during a collaboration session it all got included in the original "Information Card" molecule ticket._

## To Test
- [ ] `composer install`
- [ ] `gulp serve`
- [ ] Click "Molecules" in the SG2 menu
- [ ] Observe that you see "Information Card"
- [ ] Click "Information Card" (http://localhost:3000/?p=molecules-info-card)
  - [ ] Observe you see a heading for Card Type
  - [ ] Observe below the heading you see a gray box with a darker gray border
  - [ ] Observe in the box you see a format-able text (should show text and email linked)
  - [ ] Click the gear icon at the top right hand corner of the SG2 menu
  - [ ] Then I should see "Info Card as Media Contact" in the Variants
  - [ ] When I click "Info Card as Media Contact" 
  - [ ] Then I should be at http://localhost:3000/?p=molecules-info-card-as-media-contact 
  - [ ] And I should see an "Information Card"
  - [ ] And I should see the card type heading reads "Media Contact:"
  - [ ] And I should see a small image (max 80px) on the left side of the "Information Card"
  - [ ] And I should see the text on the right side of the "Information Card"
    - [ ] Resize your browser to `400px`
      - [ ] Observe that on smaller breakpoints the image is full width and above the text
- [ ] Click "Molecules" in the SG2 menu
- [ ] Observe that you see "Partner Information"
- [ ] Click "Partner Information" (http://localhost:3000/?p=molecules-partner-info)
  - [ ] Observe you see a logo
  - [ ] Observe below the logo you see a "Information Card as Media Contact"
  - [ ] Observe below the "Information Card as Media Contact" you see a heading 
  - [ ] Observe below the heading you see a format-able paragraph.
- [ ] Click "Organisms" in the SG2 menu
- [ ] Observe you see "Partner List"
- [ ] Click "Partner List"
  - [ ] Observe you see multiple "Partner Information" molecules
- [ ] Click "Pages" in the SG2 menu
- [ ] Click "Press Release"
- [ ] Observe you see a "Partner List" organism in the `pageContent` of the Press Release page

## Visual Regressions
I have added visual regression tests for Information Card, Info Card as Media Contact, Partner Info, and Partner List. 

### Information Card
backstop_default_molecules-info-card_0_ama__info-card_3_bp-large

#### Information Card as Media Contact
backstop_default_molecules-info-card-as-media-contact_0_ama__info-card_3_bp-large

##### Information Card as Media Contact (small)
backstop_default_molecules-info-card-as-media-contact_0_ama__info-card_0_bp-xs

### Partner Info
backstop_default_molecules-partner-info_0_ama__partner_3_bp-large

#### Partner Info (small)
backstop_default_molecules-partner-info_0_ama__partner_0_bp-xs

### Partner List
backstop_default_organisms-partner-list_0_ama__partner-list_3_bp-large

## Remaining Tasks
* [x] Add visual regression test
* [x] Add testing instructions
* [x] Add screenshot
* [x] Add partner information list to the press release page

## Additional Notes
~~Currently waiting on PR #82 to be merged so that I can add this pattern to that page.~~
~~I have intentionally _not_ updated the Glossary because I would like to discuss the name during the UX collaboration session.~~

~~In the mobile version of the wireframe it has a logo and boilerplate paired with a specific information card so that it goes logo 1, card 1, boilerplate 1, logo 2, card 2, boilerplate 2. This has some interesting implications for the data model, development and styling. I have not addressed this in the PR because we need to discuss it during the collab session.~~

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/assets/scss/03-organisms/_info-card-group.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.ama__info-card-group {
```
Comment (designerneil) [inline styleguide/source/assets/scss/03-organisms/_info-card-group.scss]: Is this class being used by anything? I can't seem to find the corresponding twig file that has the the class ama__info-card-group in it.

**Comment by designerneil** [comment]
This is an explanatory PR. In fact, I'll use it as my boilerplate.

File: styleguide/source/assets/scss/03-organisms/_info-card-group.scss (hunk 1 above)
Comment (labbydev) [reply styleguide/source/assets/scss/03-organisms/_info-card-group.scss]: Oh it is not. This has been removed. I totally forgot the CSS thanks for flagging!

**Review by chelsiejohnston — COMMENTED** [review]
@labbydev why are the images blue here? Is this still using ipsumimage?

**Review by chelsiejohnston — COMMENTED** [review]
Approve of this overall - be sure to update the ticket title to say "Partner Information" molecule vs. organism

**Pull request merged**